libc/test/src/math/smoke/CMakeLists.txt

```
@@ -3562,6 +3562,18 @@ add_fp_unittest(
     libc.src.__support.macros.properties.types
 )
 
+add_fp_unittest(
+  fmaf16_test
+  SUITE
+    libc-math-smoke-tests
+  SRCS
+    fmaf16_test.cpp
+  HDRS
+    FmaTest.h
+  DEPENDS
+    libc.src.math.fmaf16
+)
```

```suggestion
  DEPENDS
    libc.src.math.fmaf16
    libc.src.__support.cpp.type_traits
    libc.src.__support.FPUtil.cast
    libc.src.__support.macros.properties.types
)
```

The other targets FMA smoke test targets are missing some of these dependencies. Maybe there should be separate targets for these shared test headers so that we don't have to copy-paste the same dependencies several times.

---

I think if the FMA test requires functions or defined values from the DEPENDS libraries, we need to include those libraries. However, if our build is successful, it indicates that we don't actually need them, so why add them? Now I can build and `test smoke.fmaf16_test.__unit__`.